jota/src/test/java/org/iota/jota/IotaAPITest.java

```diff
@@ -78,384 +106,619 @@
-        assertTrue(res.getStates()[0], "Hash should have been seen as confirmed");
+        ArgumentException assertThrows = assertThrows(ArgumentException.class, () ->
+                iotaAPIMock.prepareTransfers(TEST_SEED2, 2, transfers, null, inputs, null, false));
+
+        assertEquals(Constants.INVALID_ADDRESSES_INPUT_ERROR, assertThrows.getMessage(), "Message should say that there is not enough balance");
     }
 
     @Test
-    @Tag("IntegrationTest")
     public void shouldFindTransactionObjects() throws ArgumentException {
-        List<Transaction> ftr = iotaAPI.findTransactionObjectsByAddresses(TEST_ADDRESSES);
-        assertThat("findTransactionObjectsByAddresses should not return null on failure", ftr, IsNull.notNullValue());
+        List<Transaction> transactions = Collections.singletonList(new Transaction.Builder()
+                .branchTransaction(BRANCH_TRANSACTION)
+                .trunkTransaction(TRUNK_TRANSACTION)
+                .address(TEST_ADDRESSES[0])
+                .signatureFragments(SIGNATURE_FRAGMENTS)
+                .tag(TEST_TAG)
+                .timestamp(Instant.now().getEpochSecond())
+                .nonce("WBTKE9RPRMHDJGITAENBMGDKWXQ")
+                .obsoleteTag(TEST_TAG)
+                .attachmentTimestamp(ATTACHMENT_TIMESTAMP)
+                .attachmentTimestampLowerBound(ATTACHMENT_TIMESTAMP_LOWER_BOUND)
+                .attachmentTimestampUpperBound(ATTACHMENT_TIMESTAMP_UPPER_BOUND)
+                .hash("AKNHNOQTIMRCETAWNHZXEHLUERYUHGCGEVFHO9JF9AL9J9CXDDUVAKBYPJMXWEJCPUJXRNXCQBRWBQ999")
+                .currentIndex(0)
+                .lastIndex(2)
+                .bundle(BUNDLE)
+                .value(1)
+                .build());
 
-        assertFalse(ftr.isEmpty(), "findTransactionObjectsByAddresses should find multiple transactions");
+        when(iotaAPIMock.findTransactionObjectsByAddresses(TEST_ADDRESSES)).thenReturn(transactions);
 
+        List<Transaction> transactionObjectsByAddresses = iotaAPIMock.findTransactionObjectsByAddresses(TEST_ADDRESSES);
+
+        assertNotNull(transactionObjectsByAddresses);
+        assertFalse(transactionObjectsByAddresses.isEmpty(), "findTransactionObjectsByAddresses should find multiple transactions");
     }
 
     @Test
-    @Tag("IntegrationTest")
-    public void shouldGetAccountData(){
-        GetAccountDataResponse gad = iotaAPI.getAccountData(TEST_SEED3, 2, 0, true, 0, true, 0, 10, true, 0);
-        assertThat("GetAccountDataResponse should not return null on failure", gad, IsNull.notNullValue());
+    public void shouldGetAccountData() {
+        when(iotaAPIMock.getAccountData(TEST_SEED3, 2, 0, true, 0, true, 0, 10, true, 0))
+                .thenReturn(new GetAccountDataResponse());
+
+        GetAccountDataResponse accountData = iotaAPIMock.getAccountData(TEST_SEED3, 2, 0, true, 0, true, 0, 10, true, 0);
+
+        assertNotNull(accountData);
```

Comment:
Codacy found an issue: [JUnit assertions should include a message](https://app.codacy.com/manual/Iota-java/iota-java/pullRequest?prid=6319461)